src/app/frontend/index.scss

```diff
@@ -18,6 +18,9 @@
 @import './variables';
 @import './mixins';
 
+///// Override styles when selected language is rtl based.
```

Comment:
Remove the comment as the file is always imported.

Reply:
Yes, It's always imported, but It only works when the language is Persian and I thought it's better to mention that.
I will remove this if it's not necessary.